mappings/net/minecraft/world/World.mapping

```diff
@@ -19,7 +23,7 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 	FIELD field_9234 thunderGradient F
 	FIELD field_9235 rainGradient F
 	FIELD field_9236 isClient Z
-	FIELD field_9238 unusedIncrement I
+	FIELD field_9238 lcgBlockSeedIncrement I
```

Comment:
Just curious, what is `lcg`?

Reply:
It's linear congruential generator.

Reply:
The abbreviation was used in existing names. I just copied the convention. If you want it documented we can do that in a separate PR.